src/__test__/integration/builtins.spec.ts

```diff
@@ -86,7 +71,7 @@ describe('Builtins usage suite', () => {
 
     // FIXME:: there is no error on broken blueprint from a node
     it.skip('create_service', async function () {
```

Comment:
fix and unskip this test
also add test with real service creation

Reply:
let's add the new test during a separate issue